app/javascript/components/common/RoundAvatar.js

```diff
@@ -3,24 +3,26 @@ import PropTypes from 'prop-types';
 import Avatar from 'antd/es/avatar';
 
 
-function RoundAvatar({ imageUrl, title, subtitle }) {
+function RoundAvatar({ logo, title, acronym }) {
```

Comment:
Not always this will be filled with logos. This component will also be used in the avatars with the picture of the party members.

I say we can keep the imageUrl, or simply image